src/css/dropdown.css

```diff
@@ -0,0 +1,209 @@
+@font-face {
+  font-family: 'anticon';
+  src: url('//at.alicdn.com/t/font_1434092639_4910953.eot');

```

Comment:
hey so we either need to get this font locally or just use the font that we have locally. The Noto-Sans

Reply:
Good catch!! let's use the one we already have